utils/file_backend_test.go

```diff
@@ -86,6 +86,24 @@ func (s *FileBackendTestSuite) TestReadWriteFile() {
 	s.EqualValues(readString, "test")
 }
 
+func (s *FileBackendTestSuite) TestCopyFile() {
+	b := []byte("test")
+	path1 := "tests/" + model.NewId()
+	path2 := "tests/" + model.NewId()
+
+	s.Nil(s.backend.WriteFile(b, path1))
+	defer s.backend.RemoveFile(path1)
+
+	s.Nil(s.backend.CopyFile(path1, path2))
+	defer s.backend.RemoveFile(path2)
+
+	_, err := s.backend.ReadFile(path1)
+	s.Error(err)
```

Comment:
This should fail...

Reply:
Interestingly, the test passes as written, and if I change this line to `s.Nil(err)`, it passes that way too. It looks like we're using the `assertion_forward.go` library incorrectly. These functions return a `bool` and have to be wrapped in an if statement. I've submitted [a separate PR to fix this entire test suite](https://github.com/mattermost/mattermost-server/pull/7871).